replicator-security/unsecure/docker-compose.yml

```diff
@@ -13,7 +13,7 @@
 ---
 version: "2.2"
 services:
-srcZookeeper:
+  srcZookeeper:
```

Comment:
This issue was introduced in 74068fdd0bf5a9a2848f775a289ece4aa2caaac2 (not clear why that change was made @rspurgeon ?)

Reply:
It's a syntax error, so clearly not intended.  Maybe it was introduced becuase of a merge conflict manual resolution typo and not caught after the merge as we don't alwasy test, or have automated tests, for merge commits.